src/library_browser.js

```diff
@@ -1305,30 +1305,33 @@ mergeInto(LibraryManager.library, {
       });
       info.awaited++;
     }
+    data = data ? Module.buffer.slice(data, data + size) : 0;
```

Comment:
need to write `Module['buffer']` so that this is ok under closure compiler advanced.